PE-463: download folder (#176)
ardriveapp/ardrive-cli: README.md

        1. [Uploading a Single File](#uploading-a-single-file)
        2. [Download a Single File (BETA)](#download-file)
        3. [Uploading a Folder with Files](#bulk-upload)
        4. [Downloading a Folder with Files](#download-folder)
        5. [Uploading Multiple Files](#multi-file-upload)
        6. [Fetching the Metadata of a File Entity](#fetching-the-metadata-of-a-file-entity)
        7. [Uploading Manifests](#uploading-manifests)
        8. [Hosting a Webpage with Manifest](#hosting-a-webpage-with-manifest)

>         9. [Create New Drive and Upload Folder Pipeline Example](#create-upload-pipeline)

Reviewer: This section `create-upload-pipeline` was removed on dev due to incompatibility with drive bundles. Probably showing up here from a merge conflict.

Author: It was because a wrong merge conflict resolution on my side. It's removed at e5bb0b9353f696fc3b526bf066a433c107967e70

ardrive upload-file --local-file-path /path/to/folder  --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

### Downloading a Folder with Files<a id="download-folder"></a>

You can download a folder on chain with the `download-folder` command.

```shell
ardrive download-folder -f "47f5bde9-61ba-49c7-b409-1aa4a9e250f6"
```

> By specifying the `--dest-folder-path` you can choose the parent folder where to download the folder on chain. When the parameter is missing, it defaults to the current working directory (`./`).

Author: ```suggestion
By specifying the `--local-path` you can choose the parent folder where to download the folder on chain. When the parameter is missing, it defaults to the current working directory (`./`).
```

```shell

> ardrive download-folder -f "47f5bde9-61ba-49c7-b409-1aa4a9e250f6" --dest-folder-path /my_ardrive_downloads/

Author: ```suggestion
ardrive download-folder -f "47f5bde9-61ba-49c7-b409-1aa4a9e250f6" --local-path /my_ardrive_downloads/
```

ardrive upload-file --local-path /path/to/folder  --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

### Downloading a Folder with Files<a id="download-folder"></a>

> You can download a folder on chain with the `download-folder` command.

Reviewer 2: ```suggestion
You can download a folder from ArDrive to your local machine with the `download-folder` command.
```

ardrive upload-file --local-path /path/to/folder  --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

### Downloading a Folder with Files<a id="download-folder"></a>

You can download a folder from ArDrive to your local machine with the `download-folder` command. In the following examples, assume that a folder with ID "47f5bde9-61ba-49c7-b409-1aa4a9e250f6" exists in your drive and is named "MyArDriveFolder".

```shell
# Downloads "MyArDriveFolder" into the current working directory, i.e. ./MyArDriveFolder/
ardrive download-folder -f "47f5bde9-61ba-49c7-b409-1aa4a9e250f6"
```

> By specifying the `--local-path` option, you can choose the local parent folder into which you the on-chain folder will be downloaded. When the parameter is omitted, its value defaults to the current working directory (i.e. `./`).

Reviewer 2: ```suggestion
By specifying the `--local-path` option, you can choose the local parent folder into which the on-chain folder will be downloaded. When the parameter is omitted, its value defaults to the current working directory (i.e. `./`).
```
